Content.Shared/CartridgeLoader/Cartridges/MessagesUiMessageEvent.cs

```diff
@@ -0,0 +1,27 @@
+using Robust.Shared.Serialization;
+
+namespace Content.Shared.CartridgeLoader.Cartridges;
+
+[Serializable, NetSerializable]
+public sealed class MessagesUiMessageEvent : CartridgeMessageEvent
+{
+    public readonly MessagesUiAction Action;
+    public readonly int? UidInput;
```

Comment:
Why is the receiver nullable? Also you should call this the receiver.

Reply:
Good point on the confusing name, I've changed it to something hopefully better.
This may have been a bad decision, but I'm using this same event for pretty much all the actions in the program UI.
This value is used to store the ID of the chat for when the user switches between chats. It can be null, because that's what the back button sets it to and because it's not always used.
The string can be null, because it is not used when switching chats.